code/game/objects/items/devices/radio/encryptionkey.dm

```diff
@@ -117,7 +117,7 @@
 /obj/item/encryptionkey/headset_mining
 	name = "mining radio encryption key"
 	icon_state = "cargo_cypherkey"
-	channels = list(RADIO_CHANNEL_SUPPLY = 1, RADIO_CHANNEL_SCIENCE = 1)
+	channels = list(RADIO_CHANNEL_SUPPLY = 1)
```

Comment:
Add //Austation removes science channel access
here

Reply:
so just add it below like i have?